src/editProfilePane/wrapped-profileFormText.ts

```diff
@@ -383,6 +395,284 @@ WHERE
+
+
+
+# was: https://solidos.github.io/solid-panes/dashboard/profileStyle.ttl#this
+# moved to:  https://solidos.github.io/profile-pane/src/ontology/profileForm.ttl#this
+#  and        https://solidos.github.io/profile-pane/src/ontology/socialMedia.ttl
+
+
+##### Ontology of Online Accounts
+
+foaf:Account a rdfs:Class;
+    rdfs:label "Online Account Provider";
+    owl:disjointUnionOf ( :BlueSkyAccount :FacebookAccount :GithubAccount :InstagramAccount
+    :LinkedInAccount :MastodonAccount :MatrixAccount 
+    :RedditAccount :SnapchatAccount :TiktokAccount  :TwitterAccount :OtherAccount) .
+
+:BlueSkyAccount rdfs:subClassOf foaf:Account ;
+    rdfs:label "Bluesky";
+    foaf:userProfilePrefix "https://bsky.app/profile/";
+    foaf:icon <https://upload.wikimedia.org/wikipedia/commons/7/7a/Bluesky_Logo.svg>;
+    foaf:homepage <https://bsky.app/> .
+
+:FacebookAccount rdfs:subClassOf foaf:Account ;
+    rdfs:label "Facebook";
+    foaf:userProfilePrefix "https://www.facebook.com/";
+    foaf:icon <https://upload.wikimedia.org/wikipedia/commons/b/b9/2023_Facebook_icon.svg>;
+    foaf:homepage <https://www.facebook.com/> .
+
+:GithubAccount rdfs:subClassOf foaf:Account ;
+    rdfs:label "Github";
+    foaf:userProfilePrefix "https://www.github.com/";
+    foaf:icon <https://upload.wikimedia.org/wikipedia/commons/9/91/Octicons-mark-github.svg>;
+    foaf:homepage <https://github.com/> .
+
+:InstagramAccount rdfs:subClassOf foaf:Account ;
+    rdfs:label "Instagram";
+    foaf:userProfilePrefix "https://www.instagram.com/";
+    foaf:icon <https://upload.wikimedia.org/wikipedia/commons/a/a5/Instagram_icon.png>;
+    foaf:homepage <https://www.instagram.com/> .
+
+:LinkedInAccount rdfs:subClassOf foaf:Account ;
+    rdfs:label "Linked In";
+    foaf:userProfilePrefix "https://www.linkedin.com/";
+    foaf:icon <https://upload.wikimedia.org/wikipedia/commons/8/81/LinkedIn_icon.svg>;
+    foaf:homepage <https://linkedin.com/> .
+
+:MastodonAccount rdfs:subClassOf foaf:Account ;
+    rdfs:label "Mastodon" ;
+    foaf:icon <https://upload.wikimedia.org/wikipedia/commons/d/d5/Mastodon_logotype_%28simple%29_new_hue.svg>;
+    foaf:homepage <https://joinmastodon.org/> .
+
+:MatrixAccount rdfs:subClassOf foaf:Account ;
+    rdfs:label "Matrix" ;
+    foaf:icon <https://upload.wikimedia.org/wikipedia/commons/7/7c/Matrix_icon.svg>;
+    foaf:userProfilePrefix "https://matrix.to/";
+    foaf:homepage <https://matrix.org/> .
+
+:MediumAccount rdfs:subClassOf foaf:Account ;
+    rdfs:label "Medium";
+    foaf:userProfilePrefix "https://www.linkedin.com/";
+    foaf:icon <https://upload.wikimedia.org/wikipedia/commons/a/a5/Medium_icon.svg>;
+    foaf:homepage <https://linkedin.com/> .
+
+:NostrAccount rdfs:subClassOf foaf:Account ; ### FEDERATED
+    rdfs:label "Nostr";
+    foaf:userProfilePrefix "https://www.linkedin.com/";
+    foaf:icon <https://upload.wikimedia.org/wikipedia/commons/b/b9/2023_Facebook_icon.svg>;
+    foaf:homepage <https://linkedin.com/> .
```

Comment:
There's no canonical homepage, but one I like is : https://nostr.net/

Reply:
Hard to know how say how to connect with you on Nostr.  Give your key, .. which relays they follow?